repose-aggregator/components/filters/keystone-v2/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Filter.scala

```diff
@@ -299,6 +304,21 @@ class KeystoneV2Filter @Inject()(configurationService: ConfigurationService,
       }
     }
 
+    def doUserTokensUpdate(userId: String, authToken: String, ttl: Int): Unit = {
+      if (CacheInvalidationFeedListener.invalidationEnabled()) {
```

Comment:
I don't feel like this particularly needs to be conditionally added, is they turn it on after the fact it'd be nice for it to start working for the items already in the cache.